src/qureg_init.cpp

```diff
@@ -35,8 +35,9 @@ QubitRegister<Type>::QubitRegister()
 
   fusion = false;
 
-  Resize(1UL);
-  state_storage[0] = {1., 0.};
+  Resize(2UL);
```

Comment:
QubitRegister::Resize() will be deprecated in the next release, but for the moment this is ok.

Always keep in mind that the argument of Resize corresponds to the new number of global amplitudes.
IQS requires at least 2 amplitudes per MPI rank so calling Resize(2) works only for 1 MPI rank.

This points to a consistent use: the argument-less constructor for QubitRegister should be used only to create a 1-qubit state in state |1> and when a single MPI rank is used.